packages/bundlers/experimental/src/ExperimentalBundler.js

```diff
@@ -1105,6 +1128,47 @@ function createIdealGraph(
       }
     }
   }
+  function attachOrphans(bundleRoot: BundleRoot) {
+    //Do not orphan anything in the bundleROOTgraph, which models parents to async and parallel deps
+    let bundleRootGraphId = bundleRootGraph.getNodeIdByContentKey(
+      bundleRoot.id,
+    );
+    let bundleId = nullthrows(bundles.get(bundleRoot.id)); // Need this for later when we delete from bundleGraph
+    let parentBundleRootIds = bundleRootGraph.getNodeIdsConnectedTo(
+      bundleRootGraphId,
+      ALL_EDGE_TYPES,
+    );
+    let parentBundleRoots = parentBundleRootIds.map(r =>
+      bundleRootGraph.getNode(r),
+    );
+    //Do not leave any orphaned children in the bundleRootGraph
+    for (let childId of bundleRootGraph.getNodeIdsConnectedFrom(
+      bundleRootGraphId,
+      ALL_EDGE_TYPES,
+    )) {
+      for (let parentId of parentBundleRootIds) {
+        if (
+          !bundleRootGraph.hasEdge(parentId, childId, 2) ||
+          !bundleRootGraph.hasEdge(parentId, childId, 1)
```

Comment:
`hasEdge` doesn't take an array or object of edge types so this is the current work around 😅  I will update this once thats fixed